_jekyll/Gemfile

```diff
@@ -28,3 +28,4 @@ end
 # Performance-booster for watching directories on Windows
 gem "wdm", "~> 0.1.1", :install_if => Gem.win_platform?
 
+gem "jekyll-minifier", "~> 0.1.10"
```

Comment:
this gem has some bugs (https://github.com/aafanasev/aafanasev.github.io/pull/9), probably has others